CodeSniffer.php

```diff
@@ -1933,6 +1933,10 @@ public static function isCamelCaps(
         $public=true,
         $strict=true
     ) {
+        if ('' === $string) {
```

Comment:
Then do it ;), because I already requested that change few inline comments above, but we're just chatting about the Yoda conditions.